specs/patterns.md

```diff
@@ -542,9 +542,58 @@ None known.
 
 A [data resource][dr] where the `profile` is the same as that of a [data package][dp] or [identifier][dpi].
 
-This could be used as part of a "Data Package Catalog", probably specified as a `profile` like
-`data-package-catalog`,`tabular-data-package-catalog`, or `document-data-package-catalog` for restricting the resources to data packages.
+### Data Package Catalog
+Data Package Resources allow for Data Package Catalogs, specified as a `profile` such as `data-package-catalog` for restricting the resources to data packages, or `tabular-data-package-catalog` for restricting the resources to tabular data packages.
```

Comment:
I'd start by explaining the motivation here - what are people trying to do ... i.e. list datasets in a data catalog.

Then introduce the solution: you provide a single Data Package that is the "Catalog" and it uses this profile.

Then each dataset in the catalog is represented in the "Catalog" Data Package by a single resource which has:

```
profile: "data-package"
format: "json"
...
```

I think we could explain things more clearly.

wdyt? And if this sounds good do you want to have a go at refactoring.

Reply:
alright, tried applying your recommendations and added a more formal "Specification" section based on tabular data packages. Not sure about it, because there isn't actually a `data-package-catalog` schema yet